components/hab/src/command/pkg/upload.rs

```diff
@@ -255,6 +260,7 @@ fn attempt_upload_dep(
     ident: &PackageIdent,
     target: &PackageTarget,
     additional_release_channel: Option<&str>,
+    _force_upload: bool,
```

Comment:
Hmm, one more q: should we also force upload the deps?

Reply:
I went back and forth on this myself. Ultimately this functionality is hopefully not exercised frequently it's mostly a safety net so I felt like defaulting to not force upload the deps was the right choice

Reply:
👍

Reply:
Hmm.  I think it makes sense to force upload the deps as well.  Otherwise, couldn't you get into a situation where a package exists but its deps don't?

Reply:
Put another way - is there a downside to force uploading the deps?